server/model/schema.json

```diff
@@ -0,0 +1,908 @@
+{
+  "basic": [
```

Comment:
I'm not sure why we are using these. It would be helpful if you could provide some comments that could benefit other developers.

Reply:
It is for the future when I will be working with MongoDB. I have just created a sample so that I don't forget what I thought at the first place.